### DISCUSSION TITLE
does dub allow changing of url?

### DISCUSSION THREAD

**Discussion by nobaraos12** [dubinc/dub#134] [Q&A] [answered]
title

**Comment by steven-tey** [comment, marked as answer]
Yup! You can edit it using our link builder:

![CleanShot 2022-12-05 at 22 38 39](https://user-images.githubusercontent.com/28986134/205816833-fa0cc8fd-1030-441d-8388-6402f6825906.png)

We have a demo on the homepage as well (you'll need to make an account to save & persist links though)

**Reply by nobaraos12** [reply]
thank you might be switching over from kutt it

**Reply by steven-tey** [reply]
Awesome! LMK if you have any questions along the way @nobaraos12!
